cloud_fixity/index.js

```diff
@@ -0,0 +1,46 @@
+const { Storage } = require('@google-cloud/storage')
+const { PubSub } = require('@google-cloud/pubsub')
+const crypto = require('crypto')
+
+exports.checkFixity = (data, context) => {
+  const attributes = JSON.parse(Buffer.from(data.data, 'base64').toString())
```

Comment:
Should error handling be introduced here if `data.data` is corrupted or `null`?

Reply:
I think it's pretty reasonable to just let it break. Would that be a problem for a reason I'm not seeing?

Reply:
I don't see it as a problem, I'm was considering the usage of `console.log` to assist with debugging these types of errors.